docs/applications/resources/models.md

```diff
@@ -41,11 +41,24 @@ Train custom TensorFlow models at scale.
     start_delay_secs: <int>  # start evaluating after waiting for this many seconds (default: 120)
     throttle_secs: <int>  # do not re-evaluate unless the last evaluation was started at least this many seconds ago (default: 600)
 
-  compute:
+  compute:         # Resources for training and evaluations steps
```

Comment:
Maybe mention it's the TensorFlow job? Like `Resources for training and evaluations steps (TensorFlow)?` Similarly `# Resources for constructing training dataset (Spark)`? We get Omer's stamp of approval on this

Reply:
@ospillinger thoughts on the comments?

Reply:
LGTM